ipn/ipnlocal/cert.go

```diff
@@ -384,6 +389,27 @@ func (s certStateStore) WriteACMEKey(key []byte) error {
 	return ipn.WriteState(s.StateStore, ipn.StateKey(acmePEMName), key)
 }
 
+// TLSCertKeyWriter is an interface implemented by stores that can write the TLS
+// cert and key in a single atomic operation. Currently this is only implemented
+// by the kubestore.StoreKube.
```

Comment:
This comment seems slightly inaccurate to me. Surely any `certStore` implementation satisfies this interface, because the function signature matches the `certStore` function signature above? Or in other words, there are multiple implementations of this interface but `kubestore.StoreKube` is the only one that writes them both atomically.

Reply:
Yeah this should have referred to 'state stores' only - I reworded it